### ISSUE TITLE
Deletion of route to 169.254.169.250 results in the route being added back immediately.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#8247]
Rancher Versions: v1.5.2-rc2

Rancher server setup with 3 hosts.

Delete the route to 169.254.169.250 from one of the host. This results in the  route getting added immediately.
```
root@sangee152rc2again-10acre-2:/home/sangeethahariharan1# ip route del 169.254.169.250 dev docker0; ip route; sleep 5; ip route
default via 10.128.0.1 dev ens4 
10.42.0.0/16 dev docker0  proto kernel  scope link  src 10.42.0.1 
10.128.0.1 dev ens4  scope link 
172.17.0.0/16 dev docker0  proto kernel  scope link  src 172.17.0.1 
default via 10.128.0.1 dev ens4 
10.42.0.0/16 dev docker0  proto kernel  scope link  src 10.42.0.1 
10.128.0.1 dev ens4  scope link 
169.254.169.250 dev docker0 
172.17.0.0/16 dev docker0  proto kernel  scope link  src 172.17.0.1 
root@sangee152rc2again-10acre-2:/home/sangeethahariharan1# 
```

```
  PID USER      PR  NI    VIRT    RES    SHR S %CPU %MEM     TIME+ COMMAND                                                                               
29162 root      20   0  254256  12864   6084 S 66.1  0.7   2:21.02 plugin-manager  
```

**Comment by leodotcloud** [comment]
Fix will be available in `rancher/network-manager:v0.5.3`(Network Services, version: v0.1.1)

**Comment by sangeethah** [comment]
Tested with - rancher/network-manager:v0.5.3 , once the route to 169.254.169.250 is deleted from the host. This results in the route getting added in the next 1 minute ( it could take anywhere between 0 - 60s) for the route to get added back.
Also plugin-manager does not not take up large CPU  like it did before.

Validation runs get done in 2 and 1/2 hours (as compared to ~4 hours it took with rancher/network-manager:v0.5.2).
